app/views/shared/_back_to_login.html.erb

```diff
@@ -0,0 +1,5 @@
+<div data-go-back-link='true' class='go-back-link' style='display:none;'>
+  <%= link_to 'javascript:;', class: "#{content_column_class} #{label_column_offset_class}", data: { no_sunet: true, target: '[data-no-sunet-content]', content_target: '[data-no-sunet-target]', link: '[data-go-back-link]' } do %>
+    <span aria-hidden=\"true\">&lsaquo; </span>Go back (show the login option)
```

Comment:
Please remove the explicit quote escaping here.  It makes the actual attribute value different:

![quote-escape](https://cloud.githubusercontent.com/assets/96776/7467822/4780ab12-f2b5-11e4-8451-a069ff53be7f.png)